zynqmp/optee-client/optee-client.bb

```diff
@@ -0,0 +1,57 @@
+OPTEE_VERSION = "3.4.0"
+SRCREV ??= "08428734c67fb559e420d87fa52fd74a955ea1bd"
+BRANCH ??= "master"
+
+DESCRIPTION = "OP-TEE Client"
+
+LICENSE = "BSD"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=69663ab153298557a59c67a60a743e5b"
+
+PROVIDES = "virtual/optee-client"
+
+inherit deploy
+
+DEPENDS += "virtual/optee-os"
+
+S = "${WORKDIR}/git"
+B = "${WORKDIR}/build"
+
+PV = "${OPTEE_VERSION}+git${SRCPV}"
+
+BRANCH ??= ""
+REPO ??= "git://github.com/OP-TEE/optee_client.git;protocol=https"
+BRANCHARG = "${@['nobranch=1', 'branch=${BRANCH}'][d.getVar('BRANCH', True) != '']}"
+SRC_URI = "${REPO};${BRANCHARG} \
+	"
+
+OPTEE_BASE_NAME ?= "${PN}-${PKGE}-${PKGV}-${PKGR}-${DATETIME}"
+OPTEE_BASE_NAME[vardepsexclude] = "DATETIME"
+
+COMPATIBLE_MACHINE = "zynqmp"
+PLATFORM_zynqmp = "zynqmp"
+
+# requires CROSS_COMPILE set by hand as there is no configure script
+export CROSS_COMPILE="${TARGET_PREFIX}"
+
+OUTPUT_DIR = "${TMPDIR}/deploy/images/${MACHINE_NAME}/optee/"
+
+EXTRA_OEMAKE_append = " CROSS_COMPILE=${CROSS_COMPILE}"
+EXTRA_OEMAKE_append = " O=${OUTPUT_DIR}"
+
+do_configure() {
+	oe_runmake clean -C ${S} clean
+}
+
+do_compile() {
+	oe_runmake -C ${S} build
+}
+
+do_install() {
+	oe_runmake -C ${S} install
+	install -d ${D}${libdir}
+	install -m 0644 ${OUTPUT_DIR}/libteec/libteec.so.1.0.0 ${D}${libdir}
+	install -d ${D}${bindir}
+	install -m 0744 ${OUTPUT_DIR}/tee-supplicant/tee-supplicant ${D}${bindir}
+	#install -d ${TMPDIR}/deploy/images/zcu102-zynqmp/optee/include
+	#install -m 0644 ${S}/public/*.h ${TMPDIR}/deploy/images/${MACHINE_NAME}/optee/include
```

Comment:
Why are those 2 commented?

Reply:
These lines can be safely removed. I will update my PR in this way.